userspace/chisel/chisel.h

```diff
@@ -18,9 +18,10 @@ limitations under the License.
 #pragma once
 
 #include <sinsp.h>
-#include "filterchecks.h"
```

Comment:
Do not include filterchecks here, forward declare the required `sinsp_filter_check` instead.